src/ur3e_hande_robot_description/urdf/hande_extension.urdf.xacro

```diff
@@ -0,0 +1,116 @@
+<?xml version="1.0"?>
```

Comment:
Explain in a comment at the top here, that this is the same as some version of [this](https://github.com/UniversalRobots/Universal_Robots_ROS2_Description/blob/humble/urdf/ur.urdf.xacro) file with added links for the hande.